core/src/main/java/hudson/model/Queue.java

```diff
@@ -1442,7 +1442,10 @@ public void maintain() {
                     if (r != null) {
                         r.run();
                     } else {
-                        new BlockedItem(top).enter(this);
+                        //this is to solve JENKINS-30084: the task has to be buildable to force
```

Comment:
mmm, right. I should probably check the reason why makeBuildable returns null somehow.